operators/olm-webhooks.adoc

```diff
@@ -0,0 +1,18 @@
+[id="olm-webhooks"]
+= Managing admission webhooks in Operator Lifecycle Manager
+include::modules/common-attributes.adoc[]
+:context: olm-webhooks
+
+toc::[]
+
+Validating and mutating admission webhooks allow Operator authors to intercept,
+modify, and accept or reject resources before they are handled by the Operator
+controller. Operator Lifecycle Manager (OLM) can manage the lifecycle of these
+webhooks when they are shipped alongside your Operator.
+
+.Additional resources
```

Comment:
Should this go at the end of the assembly? Doesn't seem like it's critical to the beginning of the assembly

Reply:
You're right, not critical, more an FYI.